src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -391,6 +206,25 @@ export default {
         this.localPools.push({ id, barcode: pool.barcode })
       })
     },
+    handleCustomProps(component) {
+      if (component.name == 'loading_target_p1_plus_p2') {
+        return {
+          ...component.props,
+          // This doesn't work at the moment as traction-input doesnt have a formatter prop
+          formatter: this.formatLoadingTargetValue,
```

Comment:
@seenanair Maybe one to look at on a Friday. The idea here is that you can pass a formatter function into an input field and it will run the function (on change i think?) to format user inputted data